guide/14-deep-learning/unsupervised-ml-model-guide.ipynb

```diff
@@ -0,0 +1,609 @@
+{
```

Comment:
<p>Data can be feature layer, spatially enabled dataframe with/without rasters or just a simple dataframe or rasters.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>